packages/components/src/components/carousel/carousel.tsx

```diff
@@ -138,6 +153,8 @@ export class Carousel {
   private btnNext: ButtonPure;
   private paginationEl: HTMLElement;
   private slides: HTMLElement[] = [];
+  private hasHeading: boolean;
+  private hasDescription: boolean;
```

Comment:
Do we need these private members? Could be local constants within the `render` lifecycle.